pkg/app/piped/cloudprovider/kubernetes/helm.go

```diff
@@ -199,3 +212,68 @@ func (c *Helm) TemplateRemoteChart(ctx context.Context, appName, appDir, namespa
 	}
 	return executor()
 }
+
+// verifyHelmValueFilePath verifies if the path of the values file references
+// a remote URL or inside the path where the application configuration file (i.e. *.pipecd.yaml) is located.
+func verifyHelmValueFilePath(appDir, valueFilePath string) error {
+	url, err := url.Parse(valueFilePath)
+	if err == nil && url.Scheme != "" {
+		for _, s := range allowedURLSchemes {
+			if strings.EqualFold(url.Scheme, s) {
+				return nil
+			}
+		}
+
+		return fmt.Errorf("scheme %s is not allowed to load values file", url.Scheme)
+	}
+
+	// absAppDir is a path where ".pipecd.yaml" is located.
+	absAppDir, err := filepath.Abs(appDir)
```

Comment:
appDir passed to this function is an absolute path so I think we don't need this.

Reply:
Removed `absAppDir` with this commit: 3ac6d692af5234cf3e62588e6b6563df73158a98